src_assets/common/assets/web/config.html

```diff
@@ -469,14 +469,20 @@ <h2 class="accordion-header">
 
         <!-- Output Name -->
         <div class="mb-3" v-if="platform === 'windows'">
-          <label for="output_name" class="form-label">Output Name</label>
-          <input type="text" class="form-control" id="output_name" placeholder="\\.\DISPLAY1"
+          <label for="output_name" class="form-label">Display Device Id</label>
+          <input type="text" class="form-control" id="output_name"
+            placeholder="\\?\DISPLAY-ACI27EC-5&4fd2de4&2&UID4355-{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}"
             v-model="config.output_name" />
           <div class="form-text">
-            Manually specify a display to use for capture. If unset, the primary display is captured.<br>
+            Manually specify a display device id to use for capture. If unset, the primary display is captured.<br>
             Note: If you specified a GPU above, this display must be connected to that GPU.<br>
-            The appropriate values can be found using the following command:<br>
-            tools\dxgi-info.exe<br>
```

Comment:
I think with this PR we can remove dxgi as it won't be needed.

Reply:
We cannot retrieve GPU names reliably with the API I am using so it is still needed, or we could just port the code, but I would like to do it in a separate PR then please.